src/main/java/com/avispl/symphony/dal/avdevices/encoderdecoder/haivision/xencoder/HaivisionXEncoderCommunicator.java

```diff
@@ -807,8 +1061,898 @@ private String replaceDoubleQuotes(String value) {
+
+				//Update bitRate with channel mode
+				String bitRate = stats.get(bitRateName);
+				String defaultBitrate = BitRateEnum.getDefaultBitRate(bitRate, value);
+
+				//default channel mode is IsStereo
+				String[] dropdownBitRate = BitRateEnum.getArrayOfNameByStereoOrMonoMode(true);
+				if (!ChannelModeEnum.STEREO.getName().equals(value)) {
+					dropdownBitRate = BitRateEnum.getArrayOfNameByStereoOrMonoMode(false);
+				}
+				AdvancedControllableProperty bitRateControlProperty = controlDropdown(stats, dropdownBitRate, bitRateName, defaultBitrate);
+				addOrUpdateAdvanceControlProperties(advancedControllableProperties, bitRateControlProperty);
+				break;
+			case APPLY_CHANGE:
+				AudioConfig audioConfig = convertAudioByValue(stats, audioName);
+
+				// sent request to apply all change for all metric
+				sendCommandToSaveAllAudioProperties(audioConfig, audioConfig.getId());
+
+				//sent request to action for the metric
+				sendCommandToSetAudioAction(audioConfig);
+				isEmergencyDelivery = false;
+				break;
+			case CANCEL:
+				isEmergencyDelivery = false;
+				break;
+			default:
+				if (logger.isDebugEnabled()) {
+					logger.debug(String.format("Controlling audio group config %s is not supported.", audioControllingMetric.getName()));
+				}
+		}
+		//Editing
+		if (isEmergencyDelivery) {
+			propertyName = audioName;
+			stats.put(propertyName + EncoderConstant.HASH + AudioControllingMetric.APPLY_CHANGE.getName(), EncoderConstant.EMPTY_STRING);
+			advancedControllableProperties.add(createButton(propertyName + EncoderConstant.HASH + AudioControllingMetric.APPLY_CHANGE.getName(), EncoderConstant.APPLY, EncoderConstant.APPLYING, 0));
+
+			stats.put(propertyName + EncoderConstant.HASH + EncoderConstant.EDITED, EncoderConstant.TRUE);
+			stats.put(propertyName + EncoderConstant.HASH + EncoderConstant.CANCEL, EncoderConstant.EMPTY_STRING);
+			advancedControllableProperties.add(createButton(propertyName + EncoderConstant.HASH + EncoderConstant.CANCEL, EncoderConstant.CANCEL, EncoderConstant.CANCELING, 0));
+		}
+	}
+
+	/**
+	 * Send command to set audio action
+	 *
+	 * @param audioConfig is instance AudioConfig DTO
+	 * @throws ResourceNotReachableException if set action audio config failed
+	 */
+	private void sendCommandToSetAudioAction(AudioConfig audioConfig) {
+		String audioId = audioConfig.getId();
+		String action = audioConfig.getAction();
+		String request = EncoderCommand.OPERATION_AUDENC.getName() + audioId + EncoderConstant.SPACE + action;
+		if (!EncoderConstant.NONE.equals(action)) {
+			try {
+				String responseData = send(request);
+				if (!responseData.contains(EncoderConstant.SUCCESS_RESPONSE)) {
+					throw new ResourceNotReachableException(String.format("Change action %s failed", audioConfig.getAction()));
+				}
+			} catch (Exception e) {
+				logger.error(String.format(EncoderConstant.COMMAND_FAILED_FORMAT, this.host, request, e));
```

Comment:
there's no need to log error separately, since exception is thrown right after this line, which will result in error logging :)